### ISSUE TITLE
[css-nesting-1] lacks examples of complex selectors

### ISSUE THREAD

**Issue by romainmenke** [w3c/csswg-drafts#7969]
This in particular seems to be missing and is important to clarify because it differs from sass :

```css
.a .b {
  .c & {}
}
```

->

```css
.c :is(.a .b) {}
```

All current examples focus on simple/compound or selector lists.

**Comment by johannesodland** [comment]
> ```css
> .a .b {
>   .c :is(.a .b) {}
> }
> ```

Did you mean to keep the nesting, or:
```css
  .c :is(.a .b) {}
```

I think it would be good to document this. I think it would be good to explain why it was designed this way as well. 

Probably this as well:
```css
.a {
  .b .c { }
}

/* equivalent to 
.a :is(.b .c) { }
*/
```

**Comment by romainmenke** [comment]
Typo, thank you @johannesodland

**Comment by romainmenke** [comment]
Your example is equivalent to 

```css
.a {
  & .b .c {}
{}
```

Which is 

```css
.a .b .c {}
```

**Comment by johannesodland** [comment]
Than. you @romainmenke . I'm getting confused 😅. Examples of complex selectors would be great.

**Comment by tabatkins** [comment]
I'm not sure I understand this issue; there *are* several examples like this in the spec, in the big block of examples in the Syntax section.

**Comment by romainmenke** [comment]
The big block in 2.1 doesn't have any examples of complex selectors in the parent 🤔.
I might be totally overlooking this.

- I made sure that I am looking at the latest version : Editor’s Draft, 27 October 2022
- I looked through everything in detail in 2.1
- I searched for `:is()`

**Comment by tabatkins** [comment]
While I don't have an example of a complex selector in the parent, I have two examples of a selector *list* in the parent, both of which explicitly show the `:is()` expansion in their "equivalent to" form. I'm not certain how your search for `:is()` didn't turn them up.

**Comment by romainmenke** [comment]
Yes, I saw those but specifically complex selectors like the one I mentioned above are note worthy. 

Lists with compound selectors work exactly the same as they do in sass in terms of matched elements. The only difference is the specificity.

Complex selectors match different elements than they would in sass and we have noticed that this is still a point of confusing for authors switching from sass to "native" nesting.
